src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -271,6 +285,13 @@ const SafeHeader = ({
           <Track {...OVERVIEW_EVENTS.OPEN_EXPLORER}>
             <StyledExplorerButton explorerUrl={getExplorerInfo(address)} />
           </Track>
+          {masterCopyError && (
+            <Tooltip title={masterCopyError}>
```

Comment:
Tooltip doesn't seem like the best choice for this.

I would opt for a snackbar, or a top bar like this:
<img width="1338" alt="Screenshot 2022-05-25 at 09 42 18" src="https://user-images.githubusercontent.com/381895/170208676-1f2ac291-2631-49de-8cf9-6a841bed3a03.png">

Reply:
I updated the Screenshot on the PR which shows an Alert in the Topbar.

Reply:
I can't see your edit 🤔

Reply:
Still fixing the tests after fixing the error with 1.1.0 contracts.